api/queries/token.query.go

```diff
@@ -1,39 +1,100 @@
 package queries
 
 import (
-	"github.com/MTthoas/dex/api/models"
-	"bytes"
 	"encoding/json"
 	"fmt"
+	"io"
 	"log"
 	"net/http"
 	"os"
+
+	"github.com/MTthoas/dex/api/models"
 )
 
-func ScrapeEthereumTokens() []models.Token {
+func GetTokens() []models.Token {
 
-	apiUrl := os.Getenv("API_URL")
+	apiUrl := "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&category=ethereum-ecosystem"
```

Comment:
Why removing it from the .env file ? Should stay there no ?

Reply:
Its only the URL, credentials are above in the header `x-cg-demo-api-key` key. 
We've have ton only change `CG_API_KEY`key in the new .env

We can still change the `vs_currency`value in the URL or put in a .env value to change currency.
But its not necessary, we only manipulates USD values for our use cases

Reply:
I think we can leave so, the url has no sensitive data